apps/web/public/static/locales/en/common.json

```diff
@@ -262,6 +262,7 @@
   "user_needs_to_confirm_or_reject_booking": "{{user}} still needs to confirm or reject the booking.",
   "user_needs_to_confirm_or_reject_booking_recurring": "{{user}} still needs to confirm or reject each booking of the recurring meeting.",
   "meeting_is_scheduled": "This meeting is scheduled",
+  "meeting_is_rescheduled": "This meeting is rescheduled",
```

Comment:
Similarly, we don't really need a special message for rescheduled bookings, just show cancel message